sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3575,15 +3568,21 @@ def run_on_head(
         log_path: str = '/dev/null',
         process_stream: bool = True,
         stream_logs: bool = False,
-        use_cached_head_ip: bool = True,
+        use_cached_head_ip: Optional[bool] = True,
         ssh_mode: command_runner.SshMode = command_runner.SshMode.
         NON_INTERACTIVE,
         under_remote_workdir: bool = False,
         require_outputs: bool = False,
         separate_stderr: bool = False,
         **kwargs,
     ) -> Union[int, Tuple[int, str, str]]:
-        """Runs 'cmd' on the cluster's head node."""
+        """Runs 'cmd' on the cluster's head node.
+
+        use_cached_head_ip: If True, use the cached head IP address. If False,
```

Comment:
Can we add other Args too? (I know, code gardening...)

Reply:
Added. The other arguments are listed in the docstr of `SSHCommandRunner.run` and `log_lib.run`. The document here is a bit duplicated. We can also change it a sentence referring to the docstr of those two functions. Wdyt?